contrib/forms/xmlformgen/xslt/report.php.xslt

```diff
@@ -107,7 +107,7 @@ $lists = array(]]></xsl:text>
               $value = $dateparts[0];
             }
 
-	    echo "<td><span class='bold'>";
+	    echo "<td style='width:24%'><span class='bold'>";
```

Comment:
If we do this in the formgen then I think it best to go ahead and wrap this in a div and give it a class. Then modify the supporting style sheet for encounter display and add the new class to style_pdf.css. Running outside 700px encounter display will mess up hover popup.
I'll look at doing in morning.